ls1intum/Artemis
# `File upload exercises:` Fix an issue with the save button

### Checklist
#### General


#### Server
#### Client
#### Changes affecting Programming Exercises

### Motivation and Context


This PR aims to fix unintended behavior during the file upload exercise creation/update. When another button is clicked in this template, it behaves as Save button and saves the changes for the exercise then navigates to exercise detail page. I removed `(ngSubmit)` for this form, and defined `click ` event for Save button as we have in other *-exercise-update templates.

### Description


### Steps for Testing

Prerequisites:
- 1 Instructor
- 1 File Upload Exercise

1. Log in to Artemis
2. Navigate to Course Management
3. Create or Find existing File Upload Exercise
4. Click Add New Criterion or Edit as Text button, you should be on the same Edit page
5. After you clicking Save button, you should see Exercise detail page.

### Review Progress





#### Code Review
#### Manual Tests

### Test Coverage





not needed

### Screenshots



<details>
  <summary>:clapper: :movie_camera: Click to see unintended behavior for File Upload Exercise</summary>  
###

https://user-images.githubusercontent.com/23650630/138928272-ec162787-9ab3-4fc8-9246-8d4b8c9b53a4.mp4  
</details>
